### PR TITLE
[Profiling] Add some code for LLaVa One Vision profiling

### PR DESCRIPTION
**Pull request by Bowen12992**
- add print info for prof
- add some default config
- fix the code indentation

### CODE REVIEW

**Review by Caozhou1995 — APPROVED** [review]
LGTM

File: flagscale/train/train.py [hunk 1]
```diff
@@ -1328,16 +1328,24 @@ def get_e2e_base_metrics():
...
+    ):
         prof = torch.profiler.profile(
-        schedule=torch.profiler.schedule(
-            wait=max(args.profile_step_start-1, 0),
-            warmup=1 if args.profile_step_start > 0 else 0,
-            active=args.profile_step_end-args.profile_step_start,
-            repeat=1),
-        on_trace_ready=torch.profiler.tensorboard_trace_handler(args.tensorboard_dir),
-        record_shapes=True,
-        with_stack=True)
```
Comment (aoyulong) [inline flagscale/train/train.py]: Please maintain these indentations to ensure consistency with Megatron-LM.

Comment (Bowen12992) [reply flagscale/train/train.py]: I tried but i cannot figure out what code format style i should obey. There are some problems with the original indentation, but the pre-commit does not have a corresponding format checks. so i just use flake8 to format my code...

**Review by Caozhou1995 — APPROVED** [review]
LGTM

**Review by aoyulong — APPROVED** [review]
LGTM

**Pull request merged**